openlibrary/plugins/openlibrary/js/carousel/Carousel.js

```diff
@@ -178,6 +180,32 @@ const Carousel = {
                 }
             });
         }
+
+        function accessibilitySetup() {
+
+            function setA11yAttributes() {
```

Comment:
Why does this need to be embedded within the function? It doesnt' seem to use the closure...